carla-carma-integration/carla-carma-agent/agent/config/sensors.json

```diff
@@ -0,0 +1,122 @@
+{
+    "sensors": [
```

Comment:
Is this configuring the sensor simulation in CARLA? If so, do these sensor positions need to be correlated with the positions in the CARMA simulation UDRF file?

Reply:
The sensor file has been modified according to the carma.urdf file